src/ui/FormatSize/FormatSize.spec.tsx

```diff
@@ -0,0 +1,60 @@
+import { render, screen } from '../../testing';
+import { FormatSize } from './FormatSize';
+
+const METRIC_UNITS = ['B', 'KB', 'MB', 'GB', 'TB', 'PB'];
+const BINARY_UNITS = ['B', 'KiB', 'MiB', 'GiB', 'TiB', 'PiB'];
+
+describe('FormatSize', () => {
+  it('Should format to Bytes', () => {
+    render(<FormatSize prefix="metric">500</FormatSize>);
+    expect(screen.queryByText('500 B')).toBeInTheDocument();
+  });
+
+  it('Should format to KB', () => {
+    render(<FormatSize prefix="metric">1000</FormatSize>);
+    expect(screen.queryByText('1 KB')).toBeInTheDocument();
+  });
+
+  it('Should format to MB', () => {
+    render(<FormatSize prefix="metric">{1000 * 1000}</FormatSize>);
+    expect(screen.queryByText('1 MB')).toBeInTheDocument();
+  });
+
+  it('Should format to float number', () => {
+    render(<FormatSize prefix="metric">1500</FormatSize>);
+    expect(screen.queryByText('1.5 KB')).toBeInTheDocument();
+  });
+
+  it('Should format to PB', () => {
+    render(
+      <FormatSize prefix="metric" metricUnits={METRIC_UNITS}>
+        {100000000 * 10000000}
+      </FormatSize>
+    );
+    expect(screen.queryByText('1 PB')).toBeInTheDocument();
+  });
+
+  it('Should format to KiB', () => {
+    render(<FormatSize prefix="binary">1024</FormatSize>);
+    expect(screen.queryByText('1 KiB')).toBeInTheDocument();
+  });
+
+  it('Should format to MiB', () => {
+    render(<FormatSize prefix="binary">{1024 * 1024}</FormatSize>);
+    expect(screen.queryByText('1 MiB')).toBeInTheDocument();
+  });
+
+  it('Should format to float number', () => {
```

Comment:
Названия повторились, по хорошему должны быть уникальны. Можно переиначить как-то так:
```suggestion
  it('Should format to float number with binary prefix', () => {
```
И аналогично для метрической.